samcli/lib/iac/cfn/helpers.py

```diff
@@ -0,0 +1,14 @@
+"""
+CFN IaC helper functions
+"""
+from samcli.lib.iac.plugins_interfaces import Stack
+from samcli.lib.providers.sam_base_provider import SamBaseProvider
+from samcli.lib.providers.sam_stack_provider import SamLocalStackProvider
+
+
+def get_resolved_cfn_template(template, parameter_overrides, global_parameter_overrides) -> Stack:
```

Comment:
nit but could we include the types for the parameters?

Reply:
Yup, good call out.